src/com/palmergames/bukkit/towny/command/PlotCommand.java

```diff
@@ -1051,10 +1053,8 @@ public void setPlotForSale(Resident resident, WorldCoord worldCoord, double forS
 				TownyMessaging.sendPrefixedTownMessage(townBlock.getTown(), Translation.of("MSG_PLOT_FOR_SALE", resident.getName(), worldCoord.toString()));
 				if (!resident.hasTown() || (resident.hasTown() && townBlock.getTown() != resident.getTown()))
 					TownyMessaging.sendMessage(resident, Translation.of("MSG_PLOT_FOR_SALE", resident.getName(), worldCoord.toString()));
-				Bukkit.getPluginManager().callEvent(new PlotSetForSaleEvent(resident, forSale, townBlock));
 			} else {
 				TownyMessaging.sendMessage(resident, Translation.of("msg_plot_set_to_nfs"));
-				Bukkit.getPluginManager().callEvent(new PlotNotForSaleEvent(resident, townBlock));

```

Comment:
Looks like you're not on an up to date master branch, so if I accept this it will remove the new forsale/notforsale events.